### PR TITLE
Vault 1.5.3 ubi8 image update

### PR DESCRIPTION
**Pull request by snehakpersistent**
Updated Redhat ubi8 image for Vault 1.5.3
Installed upgraded version of libgomp required by gcc

### CODE REVIEW

**Comment by seth-priya** [comment]
@kandarpamalipeddi please review

File: v/vault/Dockerfiles/1.5.3-ubi8/README.md [hunk 1]
```diff
@@ -7,3 +8,38 @@ Docker run command:
...
+    You can add applications to this project with the 'new-app' command. For example, try:
+    oc new-app rails-postgresql-example
+    to build a new example application in Ruby. Or use kubectl to deploy a simple Kubernetes application:
+    kubectl create deployment hello-node --image=k8s.gcr.io/serve_hostname
+    
+$ oc adm policy add-scc-to-user anyuid -z default
+--> clusterrole.rbac.authorization.k8s.io/system:openshift:scc:anyuid added: "default"
+
+$ oc new-app ibmcom/vault-ppc64le:1.5.3-ubi8 -e SKIP_CHOWN=true -e SKIP_SETCAP=true
+--> Found container image 6e8b586 (6 hours old) from quay.io for "quay.io/snehakpersistent/vault:latest"
```
Comment (kandarpamalipeddi) [inline v/vault/Dockerfiles/1.5.3-ubi8/README.md:24]: Please please recheck the image name ?
app is being created with `ibmcom/vault-ppc64le:1.5.3-ubi8` image, but in comment it was mentioned as `quay.io`

File: v/vault/Dockerfiles/1.5.3-ubi8/README.md [hunk 2]
```diff
@@ -7,3 +8,38 @@ Docker run command:
         Server mode:
         docker run --cap-add=IPC_LOCK -e 'VAULT_LOCAL_CONFIG={"backend": {"file": {"path": "/vault/file"}}, "default_lease_ttl": "168h", "max_lease_ttl": "720h"}' ppc64le/vault server
 
+# Deploying Vault on Openshift
+```
+$ oc new-project vault-test
+--> Now using project "vault-test" on server "https://api.sneha-ppc.cp.fyre.ibm.com:6443".
```
Comment (kandarpamalipeddi) [inline v/vault/Dockerfiles/1.5.3-ubi8/README.md:14]: Can you replace the server address with `https://api-<cluster-address>:6443` ?
As no one can access the address as you are not providing username & password.

File: v/vault/Dockerfiles/1.5.3-ubi8/Dockerfile [hunk 3]
```diff
@@ -15,8 +15,8 @@ ENV PATH $PATH:$GOPATH/bin
 
 # Set up our base tools, and Vault.
 RUN set -eux; \
-    yum install -y openssl sudo \ 
-    make git gcc wget && \
+    yum install -y https://www.rpmfind.net/linux/centos/8-stream/BaseOS/ppc64le/os/Packages/libgomp-8.4.1-1.el8.ppc64le.rpm \
```
Comment (kandarpamalipeddi) [inline v/vault/Dockerfiles/1.5.3-ubi8/Dockerfile:18]: Can you please explain the requirement of `libgomp`?

File: v/vault/Dockerfiles/1.5.3-ubi8/README.md (hunk 1 above)
Comment (snehakpersistent) [reply v/vault/Dockerfiles/1.5.3-ubi8/README.md]: Updated it to ibmcom/vault-ppc64le:1.5.3-ubi8. I was using the image from quay for testing purpose as the new image is not yet pushed to ibmcom.

File: v/vault/Dockerfiles/1.5.3-ubi8/README.md (hunk 2 above)
Comment (snehakpersistent) [reply v/vault/Dockerfiles/1.5.3-ubi8/README.md]: Updated.

File: v/vault/Dockerfiles/1.5.3-ubi8/Dockerfile (hunk 3 above)
Comment (snehakpersistent) [reply v/vault/Dockerfiles/1.5.3-ubi8/Dockerfile]: libgomp-8.4.1 is required by gcc 8.4.1. The yum repository was installing lower version of libgomp and hence the gcc installation was failing. 
I believe, this change is no more required. Looks like yum repo has the upgraded version now. Have reverted this change.

**Comment by snehakpersistent** [comment]
@kandarpamalipeddi since the libgomp change is removed, there is no change required in Dockerfile. Do you need the updated README? If not, I believe we can close this issue.
@seth-priya ^^

**Review by kandarpamalipeddi — APPROVED** [review]

**Comment by kandarpamalipeddi** [comment]
@seth-priya 
Changes looks good to me.

**Pull request merged**